docs/deploy-hyperlane-agents.mdx

```diff
@@ -1,177 +0,0 @@
-# Get Started
```

Comment:
It's not currently referenced, but IMO we should still have this option open for folks who do not want to use Kurtosis

Reply:
Both pages have a lot of overlap and we already have a callout on the main page that points to the non-kurtosis validator/relayer docs. I could bump that up to the start of the "run validator/relayer" step and make it more obvious so that users know from the start that they run it themselves?